src/powershell/set-wallpaper.ps1

```diff
@@ -0,0 +1,32 @@
+Write-Output "[ ] Attempting to set Commando VM wallpaper"
+
+$path = $args[0]
+$setwallpapersrc = @"
```

Comment:
This inline code could use some documentation since it's using hard-coded values and directly interfacing with a system DLL.

Reply:
@mcdonnnj resolved with [d2e72cb](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/d2e72cb9082563fc8f806afa594764780519d98a)

Reply:
This change is less helpful than what it replaced because now it's even more opaque as to what it's doing. I was looking for a link to documentation around the arguments being passed, not for you to remove the usefully named variables and just move to hard-coded values in the method call.

Reply:
Unfortunately, those named variables didnt work. This was a bug that I fixed by removing them. I will have a more descriptive comment shortly

Reply:
I added a detailed comment explaining each argument being passed. Resolved in [14745f0](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/14745f056038740507380b6b9ebd7d9d5a0e534b)

Reply:
@mcdonnnj I added a detailed comment explaining each argument being passed. Resolved in [14745f0](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/14745f056038740507380b6b9ebd7d9d5a0e534b)